tests/test_make.py

```diff
@@ -1467,6 +1467,15 @@ class C2(C):
         assert [C2] == C.__subclasses__()
 
 
+    def test_cache_hash_with_frozen_serializes(self):
+        """Frozen classes with cache_hash should be serializable."""
+        @attr.s(cache_hash=True, frozen=True)
+        class C:
```

Comment:
Unfortunately, we still support Python 2.7. :( This is the reason the 2.7 builders broke.